packages/sdk-v2/src/sdk/claim/react.ts

```diff
@@ -15,11 +15,14 @@ import useRefreshOrNever from "../../hooks/useRefreshOrNever";
 import { useGetContract, useGetEnvChainId, useReadOnlySDK, useSDK } from "../base/react";
 import { Envs, SupportedChains } from "../constants";
 
-export const useFVLink = () => {
-  const { chainId } = useGetEnvChainId();
+export const useFVLink = (requiredChainId?: number, whitelistAtChain: boolean = false) => {
```

Comment:
I dont understand this, this should simply be "chainId"
no need for whitelistAtChain or useGetEnvChainId

Reply:
@sirpy useGetEnvChainId was before, to obtain default chainId for useSDK() hook
I kept it to fall back to this value if no chain id been passed